stubs/Parsers/ArgumentsParser.swift

```diff
@@ -65,6 +67,17 @@ class ArgumentsParser {
     }
 
     private func actualFilePath(for pathArgument: String, using executableFolderPathURL: URL, isDirectory: Bool) -> URL? {
+        // Absolute path
+        if pathArgument.prefix(1) == "/" {
```

Comment:
This helped with debugging. Should make the script a bit more flexible as it doesn't assume that the stub path is relative to where the script is located.

Reply:
Yes, this is a better way, but the code below was checking both options as well.